### PR TITLE
DOCSP-37163 Remove Outdated Electron Guides from TOC

### PR DESCRIPTION
**Pull request by lindseymoore**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-37163

- [Node](https://preview-mongodblindseymoore.gatsbyjs.io/realm/DOCSP-37163/sdk/node/): removed Integration Guides section from TOC.

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [ ] Did you tag pages appropriately?
  - genre
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [ ] Create a Jira ticket for related docs-app-services work, if any

### Release Notes

**Node.js SDK**
- Integration Guides: Remove Integration Guides section from TOC, as these electron guides are outdated.

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: config/redirects [hunk 1]
```diff
@@ -1285,3 +1285,8 @@ raw: ${prefix}/sdk/flutter/realm-database/read-and-write-data -> ${base}/sdk/flu
 # Smartling missing redirect
 
 raw: ${prefix}/sdk/react-native/use-realm-react -> ${base}/sdk/react-native/api-reference/realm-provider/
+
+# DOCSP-37163 Remove Electron Guides from TOC
+
+raw: ${prefix}/sdk/node/integrations/electron-cra -> ${base}/sdk/node/
```
Comment (cbullinger) [inline config/redirects:1291]: do we need to include the [integrations landing page](https://www.mongodb.com/docs/realm/sdk/node/integrations/) too?
```suggestion
raw: ${prefix}/sdk/node/integrations -> ${base}/sdk/node/
raw: ${prefix}/sdk/node/integrations/electron-cra -> ${base}/sdk/node/
```

File: config/redirects [hunk 2]
```diff
@@ -1285,3 +1285,8 @@ raw: ${prefix}/sdk/flutter/realm-database/read-and-write-data -> ${base}/sdk/flu
 # Smartling missing redirect
 
 raw: ${prefix}/sdk/react-native/use-realm-react -> ${base}/sdk/react-native/api-reference/realm-provider/
+
+# DOCSP-37163 Remove Electron Guides from TOC
```
Comment (cbullinger) [inline config/redirects:1289]: I can still get to these pages via direct url (e.g. [Electron Guide](https://preview-mongodblindseymoore.gatsbyjs.io/realm/DOCSP-37163/sdk/node/integrations/electron/)) - do these redirects not kick-in on staged builds?

**Review by cbullinger — APPROVED** [review]
LGTM assuming there aren't any linked refs to these now-hidden guides that should also be updated/removed (keeping the reference/link and redirecting to main Node page seems like a bad UX?)

File: config/redirects (hunk 2 above)
Comment (krollins-mdb) [reply config/redirects:1289]: Yeah, redirects only work on the published site.

**Comment by lindseymoore** [comment]
No additional refs to these pages

**Pull request merged**